src/main/java/PeselValidator.java

```diff
@@ -0,0 +1,134 @@
+
+public class PeselValidator {
+
+
+    public boolean validate(String pesel) {
+
+        if (!checkPeselLength(pesel)) {
+            return false;
+        }
+
+        if (!checkPeselType(pesel)) {
+            return false;
+        }
+
+        if (!checkPeselMonth(pesel)) {
+            return false;
+        }
+
+        if (!checkPeselDay(pesel)) {
+            return false;
+        }
+
+        if (!checkPeselLastNumber(pesel)) {
+            return false;
+        }
+
+
+        return true;
+    }
+
+
+    private boolean checkPeselLength(String pesel) {
+
+        if (pesel.length() != 11) {
+            throw new NumberFormatException();
+
+        } else {
+            return true;
+        }
+    }
+
+    private boolean checkPeselType(String pesel) {
+        String firstHalfPeselStr = pesel.substring(0, 5);
+        String secondHalfPeselStr = pesel.substring(5, 11);
+
+        try {
+            int firstHalfPesel = Integer.valueOf(firstHalfPeselStr);
+            int secondHalfPesel = Integer.valueOf(secondHalfPeselStr);
+
+        }catch (NumberFormatException e) {
+            System.out.println("Incorrect type.");
+        }
+
+        return true;
+    }
+
+
+    private boolean checkPeselDay(String pesel) {
+        String peselDay = pesel.substring(4, 6);
+        int peselDayOfMonth = Integer.valueOf(peselDay);
+
+        if (peselDayOfMonth > 12) {
+            return false;
+        }
+
+        return true;
+    }
+
+    private boolean checkPeselMonth(String pesel) {
+        String stringPeselMonth = pesel.substring(2, 4);
+
+        int intPeselMonth = Integer.valueOf(stringPeselMonth);
+
+
+        if (intPeselMonth > 12 && intPeselMonth < 21) {
+            return false;
+
+        }else if (intPeselMonth > 32 && intPeselMonth < 41) {
```

Comment:
Space is missing here and in other places -> please use automatic IntelliJ tool to reformat code (Code->ReformatCode)